crates/phactory/api/Cargo.toml

```diff
@@ -29,6 +29,8 @@ anyhow = { version = "1.0.43", optional = true }
 log = { version = "0.4.14", optional = true }
 reqwest = { version = "0.11.4", optional = true }
 
+primitive-types = { version = "0.10.1", optional = true, default-features = false }
```

Comment:
should we use `sp-io` instead of `primitive-types`? or we may meet version conflict (we've met this before, sp-io re-expose primitive-types)

Reply:
We can't. The rust code itself has not direct dependency on primitive-types. The purpose we add it here is to turn on it's serde feature.
Doesn't the version conflict is easy to be solved?

Reply:
ok